server/utils/referralUtils.test.ts

```diff
@@ -101,7 +102,7 @@ describe('ReferralUtils', () => {
                 text: 'cannot start yet',
               },
               text: 'Check answers and submit',
-              url: '#',
+              url: referPaths.checkAnswers({ referralId: referral.id }),
```

Comment:
Let's (mostly) hardcode this in the test but still use the path method in the utils itself. So here: `/referrals/${referral.id}/checkAnswers` (we tend to do this in tests on code that use path methods)